includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -82,7 +82,7 @@ abstract class AMP_Base_Sanitizer {
 	 *
 	 * @var array
 	 */
-	private $should_not_removed_nodes = [];
+	private $nodes_to_keep = [];
```

Comment:
😜

Reply:
Oh, maybe I should mention I searched for usage first:
- https://wpdirectory.net/search/01DV66Q5TGAA54FBQJA46N9D6R

![Image 2019-12-03 at 4 43 35 PM](https://user-images.githubusercontent.com/83631/70065862-0f37c600-15ec-11ea-974f-954cadb5480b.png)